pages/content/pixi/recommendations/no-icon-fonts.md

```diff
@@ -0,0 +1,8 @@
+---
+$title: Don't use icon fonts
+$order: 200
+tags:
+  - cls
+  - lcp
+---
+Icon fonts have a negative impact on CLS, causing layout shift as they load in (unless blocked from loading). Document authors are unaware of the impact since it rarely appears on high-speed connections.
```

Comment:
`Icon fonts negatively affect accessibility and load times of your page. They also can cause content shifts on load. Consider using SVGs as an alternative instead to improve LCP and CLS.`

Reply:
I would move the ```also``` after the ```can```, so:
```... They can also cause content shifts on load.```

Reply:
Hm. Maybe we should make it even simpler:

```
Icon fonts negatively affect accessibility and performance of your page. Consider using SVGs as an alternative instead to improve LCP and CLS.
```

Reply:
Perfect. 👍🏻